pkg/metrics/project.go

```diff
@@ -48,4 +59,63 @@ func (c gardenMetricsCollector) collectProjectMetrics(ch chan<- prometheus.Metri
 		}
 		ch <- metric
 	}
+
+	// Determine user counts.
+	var (
+		metric prometheus.Metric
+
+		users          = make(map[string]bool)
+		groups         = make(map[string]bool)
+		technicalUsers = make(map[string]bool)
+	)
+
+	for _, project := range projects {
+		for _, user := range project.Spec.Members {
+
+			// Check if user is of kind service account.
```

Comment:
Do you need to check for a service account here if you check in the next switch?